### PR TITLE
chore(ci): kill all nodes before packaging logs to upload

### PR DESCRIPTION
**Pull request by bochaco**

### CODE REVIEW

File: .github/workflows/sn_pr_tests.yml [hunk 1]
```diff
@@ -411,6 +420,15 @@ jobs:
   #         echo "$(pgrep sn_node | wc -l) nodes still running"
   #         ls $HOME/.safe/node/local-test-network
 
+  #      - name: Kill all nodes
```
Comment (joshuef) [inline .github/workflows/sn_pr_tests.yml:423]: lets remove this if we're doing it above then :+1

**Review by joshuef — CHANGES_REQUESTED** [review]

File: .github/workflows/sn_pr_tests.yml (hunk 1 above)
Comment (bochaco) [reply .github/workflows/sn_pr_tests.yml:423]: do you mean in the previous step? that's only listing them...or what do you mean?

Comment (joshuef) [reply .github/workflows/sn_pr_tests.yml:423]: i just meant the commented code for `kill all nodes`.

Comment (bochaco) [reply .github/workflows/sn_pr_tests.yml:423]: But the earlier one is for the other job, i.e. `e2e` job, this one is for whenever we re-enable `e2e-split` job, sorry, but I don't think I understand?

Comment (joshuef) [reply .github/workflows/sn_pr_tests.yml:423]: Ahhh, no. I don't think _i_ understood. All good then

**Review by joshuef — APPROVED** [review]

**Pull request merged**